rules_poetry/poetry.bzl

```diff
@@ -79,8 +94,31 @@ def _mapping(repository_ctx):
     return {
         "dependencies": dependencies,
         "groups": groups,
+        "pyproject": pyproject
     }
 
+def extract_markers(repository_ctx, resolved_markers, dep, markers):
+    python_interpreter = _resolve_python_interpreter(repository_ctx)
+
+    if str(type(markers)) == 'list':
+        if dep not in resolved_markers:
+            resolved_markers[dep] = {}
+        for marker in markers:
```

Comment:
Using `items()` here will allow you to use `version` directly, so you won't need to use `marker['version']` multiple times

Reply:
actually in this case markers is supposed to be a list, so I made some changes around this, but the specific change you requested is not there [7aa2a09](https://github.com/mongodb-forks/rules_poetry/pull/1/commits/7aa2a09847cc9482b937f4381021a9cb45946821)